slm_lab/agent/algorithm/actor_critic.py

```diff
@@ -68,13 +68,11 @@ def post_body_init(self):
 
     @lab_api
     def body_act_discrete(self, body, state):
-        '''Returns a discrete action'''
-        return self.action_policy(self, state)
+        return self.action_policy(self, state, body, self.agent)
```

Comment:
can internalize the `self.agent` arg to internal using the `self` already passed in as the first paramarg

Reply:
Good point. Fixed